### ISSUE TITLE
when doing --fix async functions are not fixed properly as the async is removed after the fix

### ISSUE THREAD

**Issue by immortal-loop** [markalfred/eslint-plugin-no-constructor-bind#40]
Hi Mark,

there is an issue that I am facing while using --fix command

```
// INPUT
class Person {
    constructor(props) {
        super(props)
        this.walk = this.walk.bind(this)
    }
    async walk() { }
}

// OUTPUT
class Person {
    class Person {
    constructor(props) {
        super(props)
    }
    walk = () => { }
}

// OUTPUT EXPECTED
class Person {
    class Person {
    constructor(props) {
        super(props)
    }
    async walk = () => { }
}
```

Please help

**Comment by markalfred** [comment]
Thank you Vikas, this should be fixed in [v2.0.1](https://github.com/markalfred/eslint-plugin-no-constructor-bind/releases/tag/v2.0.1). Hope this didn't cause any headaches for you and thanks for using the plugin!

**Comment by immortal-loop** [comment]
thanks, Mark, I am enjoying this plugin. However, I have got another issue while using the v2.0.1

As shown in the scene shot below after running the fix
1. not all bind function got converted to the arrow functions.
2. line 57 and 58 there are semicolon are left.

<img width="1671" alt="Screenshot 2020-06-17 at 6 12 54 PM" src="https://user-images.githubusercontent.com/66246267/84899343-2ccf0780-b0c6-11ea-8890-5ad4067b955a.png">
